examples/coagents-starter/agent-js/src/agent.ts

```diff
@@ -49,12 +52,10 @@ async function chat_node(state: AgentState, config: RunnableConfig) {
 
   // 6.2 Bind the tools to the model, include CopilotKit actions. This allows
   //     the model to call tools that are defined in CopilotKit by the frontend.
-  const modelWithTools = model.bindTools!(
-    [
-      ...(state.copilotkit?.actions ?? []),
-      ...tools,
-    ],
-  );
+  const modelWithTools = model.bindTools!([
```

Comment:
idk what y'all are using for code formatting but I'm using prettier and it opines that this should be formatted like this